avro-builder.gemspec

```
@@ -15,6 +15,12 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/salsify/avro-builder.git'
   spec.license       = 'MIT'
 
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = 'https://rubygems.org'
```

Would be good to add the new `rubygems_mfa_required` too: https://guides.rubygems.org/mfa-requirement-opt-in/

```suggestion
    spec.metadata['allowed_push_host'] = 'https://rubygems.org'
    spec.metadata['rubygems_mfa_required'] = 'true'
```

---

The intent behind this is to support our internal automation that will be managing Gem releases moving forward (vs engineers). This would in turn break that automation.

It does bring up a good point that maybe certain libraries (where we have external collaborators with the ability to push) should be excluded from this automation.

What are your thoughts @jturkel ?

---

I confirmed setting the MFA scope to `UI and Gem Signin` allowed me to push a Gem twice with an API token:

- Once to turn this one
- Second one, per the docs, which also worked